### PR TITLE
Agenda no events msg

### PR DESCRIPTION
**Pull request by mulholio**
#### Solution to https://github.com/intljusticemission/react-big-calendar/issues/922

When in the agenda view, you often come across empty tables of events. This looks unfinished and is confusing to the user as it may appear that data has not loaded, rather than there is simply no data to display.

The solution is to display a message, informing the user that there are no events and the component is functioning correctly.

I think there may be a little bit of work to do in terms of styling/aligning the message. That said, I imagine most people are going to be adding their own project styles and therefore I feel it's okay to leave the styling basic and unopinionated.

### CODE REVIEW

File: src/Agenda.js [hunk 1]
```diff
@@ -48,26 +48,34 @@ class Agenda extends React.Component {
...
+              <table className="rbc-agenda-table">
+                <tbody ref="tbody">
+                  {range.map((day, idx) => this.renderDay(day, events, idx))}
+                </tbody>
+              </table>
+            </div>
+          </React.Fragment>
+        ) : (
+          <span className="rbc-agenda-empty">
+            There are no events in this range
```
Comment (jquense) [inline src/Agenda.js]: this needs to be configurable. It should go in `messages` and added to the top level calendar prop-type as well

Comment (jquense) [reply src/Agenda.js]: we should also add some default styling

Comment (miltonmc) [inline src/Agenda.js]: Since we can configure other labels, I think it should be configurable as well. 
@jquense what do you think about that?

Comment (jquense) [reply src/Agenda.js]: https://github.com/intljusticemission/react-big-calendar/pull/923#discussion_r205815806

**Comment by mulholio** [comment]
Had another look and can't work out/decided the best place to put the configuration option. Would it be possible to give some pointers? Would I be right in thinking this goes in either `messages` or `components` props?

**Comment by mulholio** [comment]
Have attempted to add `noEventsInRange` into the message `Calendar` props but seem to be getting a bug/issue with the month view not displaying properly (see below) when I try to test with an override message.

screen shot 2018-07-30 at 11 32 44

**Comment by jquense** [comment]
@mulholio can you add a storybook story that demonstrates the issue? I can take a look at that

**Pull request merged**